servicebroker/test/ut/api/api_test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+var should = require('should');
+var API = require(path.join(__dirname, '../../../lib/api/api.js'));
+
+var api = new API();
+
+describe('Unit Test for API', function() {
+
+  it('getCatalog() should return catalog info with JSON format', function() {
+    var data = api.getCatalog();
+    data.should.have.property('services');
```

Comment:
yes, it should match the content of catalog.json.   
Current unit test purpose is just verify the getCatalog() function return a valid JSON format, and include service definition inside. 
Do you mean we need to parse catalog.json file here, and make sure data=catalog exactly in unit test?